libs/databrickscfg/ops.go

```diff
@@ -130,17 +129,20 @@ func SaveToProfile(ctx context.Context, cfg *config.Config) error {
 	return configFile.SaveTo(configFile.Path())
 }
 
-func ValidateConfigAndProfileHost(cfg *databricks.Config, profile string) error {
+func ValidateConfigAndProfileHost(cfg *config.Config, profile string) error {
 	configFile, err := config.LoadFile(cfg.ConfigFile)
 	if err != nil {
+		if os.IsNotExist(err) {
+			return nil
```

Comment:
Not entirely clear what happens if configuration file is not found, are we just skipping this validate and fail at a later point?

Reply:
Good point -- it did indeed skip validation, similar to how the SDK allows a profile to be set but doesn't error:

https://github.com/databricks/databricks-sdk-go/blob/415522e1015ea099b775ae1a89e8b5721dcc2168/config/config_file.go#L79-L83

However, the intent here is always to use a profile if it is specified, so it should return an error if we cannot resolve the profile. Addressed this in the next commit.